server/lib/scene/scene.get.js

```diff
@@ -58,14 +59,25 @@ async function get(options) {
   }
 
   if (optionsWithDefault.searchTags) {
-    const tags = await db.TagScene.findAll({
-      fields: 'scene_id',
-      where: {
-        [Op.or]: optionsWithDefault.searchTags.split(',').map((tag) => ({ name: { [Op.like]: `%${tag}%` } })),
-      },
-    });
+    const tags = optionsWithDefault.searchTags.split(',');
+
+    const tagsWithSceneId = await Promise.all(
```

Comment:
Can you do this in one query like before?

Reply:
If it's or needed no, if it's and needed, I come back to the previous code

Reply:
Finally, I found a solution with only 1 query but I use more the cpu for the calculation :)